src/FixedDataTableRow.js

```diff
@@ -170,16 +170,17 @@ class FixedDataTableRowImpl extends React.Component {
       return true;
     }
 
+    // if row is not visible then no need to render it
+    // change in visibility is handled by the parent
+    if (!nextProps.visible) {
```

Comment:
moved this check above since this can serve as an early skip for re-rendering rows.

Reply:
But this way we skip updating when row visibility changes. have you tested this won't lead to any issue ?.

Reply:
We only skip re-renders if the row is hidden.
This should be fine since the parent (rowWrapper) actually handles the style updates for hiding the row.

I smoke tested on our examples, and they seem to work fine.